src/openzaak/components/catalogi/admin/forms.py

```diff
@@ -83,25 +82,6 @@ def has_changed(self):
     def clean(self):
         super().clean()
 
```

Comment:
Is this all fixed in upstream django-better-admin-arrayfield?

Reply:
Yep! Test coverage dropped because these lines were no longer hit, and I verified it manually + made the tests more explicit